src/jstests/formio/components/multiple.spec.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import _ from 'lodash';
+import {Formio} from "react-formio";
+import {multipleForm} from './fixtures/multiple';
```

Comment:
```suggestion

import {multipleForm} from './fixtures/multiple';
```

I'm trying to group library vs. own-code imports :)